example_scripts/galaxy_zoo_touch_table/prepare_db_from_classification_export.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import pandas as pd
+import json
+import string
+from tqdm import tqdm
+from itertools import cycle
+
+def preprocessing():
+    # Read the local CSV subject export
+    classifications=pd.read_csv("classification-export.csv")
+
+    classifications['metadata']=classifications['metadata'].map(lambda x: json.loads(x))
+    classifications['locations']=classifications['locations'].map(lambda x: json.loads(x))
+
+    # Include in subject_set_ids all subject sets you want to keep
+    subject_set_ids = [];
+    classifications = classifications.loc[classifications['subject_set_id'].isin(subject_set_ids)]
```

Comment:
Kudos on using `.loc` correctly. A lot of times I see slicing done for something like this without it, which creates a copy of the dataframe rather than a window into the original one.

Reply:
The aforementioned warning, explained with pictures that make it really clear what this is: https://www.dataquest.io/blog/settingwithcopywarning/